src/get-started/flutter-for/ios-devs.md

```diff
@@ -5,467 +5,838 @@ description: Learn how to apply iOS developer knowledge when building Flutter ap
+you will get an overflow warning, as shown below:
 
-The following example shows how to use a `StatelessWidget`.
-A common`StatelessWidget` is the `Text` widget.
-If you look at the implementation of the `Text` widget,
-you'll find it subclasses `StatelessWidget`.
 
-<?code-excerpt "lib/text_widget.dart (TextWidget)" replace="/return const //g"?>
 ```dart
-Text(
-  'I like Flutter!',
-  style: TextStyle(fontWeight: FontWeight.bold),
-);
+UnconstrainedBox(
+  child: Container(color: red, width: 4000, height: 50),
+)
 ```
 
-If you look at the code above, you might notice that the `Text` widget
-carries no explicit state with it. It renders what is passed in its
-constructors and nothing more.
+<img src="/assets/images/docs/ui/layout/layout-14.png" alt="When parents pass unbounded constraints to children, and the children are expanding, then there is an overflow warning">
 
-But, what if you want to make "I Like Flutter" change dynamically,
-for example when clicking a `FloatingActionButton`?
+For more information on how constraints work in Flutter, 
+see [Understanding constraints][]. 
 
-To achieve this, wrap the `Text` widget in a `StatefulWidget` and
-update it when the user clicks the button.
 
-For example:
+### Design system
 
-<?code-excerpt "lib/text_widget.dart (StatefulWidget)"?>
-```dart
+Because Flutter is multi-platform, your app doesn’t need to conform to a stock design system. 
+While many examples in the Flutter documentation feature Material widgets, 
+which leverage the [Material Design guidelines][], you have a lot of options 
+for designing your Flutter app.  You can customize Material widgets, leverage 
+widgets built by the community, build your own widgets, 
+or use the [Cupertino widgets][] that follow Apple’s Human Interface Guidelines.
 
-class SampleApp extends StatelessWidget {
-  // This widget is the root of your application.
-  const SampleApp({super.key});
+<iframe width="560" height="315" src="{{site.youtube-site}}/embed/3PdUaidHc-E?rel=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-  @override
-  Widget build(BuildContext context) {
-    return const MaterialApp(
-      title: 'Sample App',
-      home: SampleAppPage(),
-    );
+If you're looking for a great reference app that features a custom design system, check out [Wonderous][].
+
+## UI Basics
+
+This section covers how basic UI components in Flutter compare to those in SwiftUI. 
+This includes how to start your app, display static text, 
+create buttons, react to on-press events, display lists, grids, and more.
+
+
+### Getting started
+
+Usually, when you implement your SwiftUI app, you begin with **App:**
+
+```swift
+@main
+struct MyApp: App {
+    var body: some Scene {
+        WindowGroup {
+            HomePage()
+        }
+    }
+}
+```
+
+The body is usually created in a struct that conforms to the **View** protocol as follows:
```

Comment:
**issue:** See previous.
```suggestion
Write the app body within a `struct` that conforms to the **View** protocol as follows:
```

Reply:
Same comment as above, I don't want to tell people how to write SwiftUI code, we're simply comparing common ways of doing things in SwiftUI to how we recommend it in Flutter

Reply:
Gotcha. Maybe:

```markdown
Another common SwiftUI development practice places the app body with a `struct`. This conforms...
```

Reply:
Added!